features-json/webauthn.json

```diff
@@ -296,7 +296,8 @@
       "46":"n"
     },
     "and_chr":{
-      "69":"n"
+      "69":"n",
+      "70":"y"
```

Comment:
This PR is invalid because it adds new browser version

Reply:
So should it just be `"70":"y"` instead?

Reply:
Nothing. Just remove it.

Reply:
I'm confused, how does this help in signalling that a soon-to-be-released version has support? That's what I was trying to accomplish.

I've run the tests locally (they passed) and I can't find anything in CONTRIBUTING.md about multiple versions being okay for `chrome` but not for `and_chr`? I must be missing something...

Reply:
No. It is for all browsers

Reply:
Yeah, currently upcoming versions for Android aren't tracked. If you'd like to mention that it's coming you can add it in the `notes` section.